etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json

```diff
@@ -1615,11 +1615,11 @@
                 "datasource": "",
                 "hide": 0,
                 "includeAll": false,
-                "label": "Cluster Id",
+                "label": "Cluster Name",
                 "multi": false,
                 "name": "cluster_id",
                 "options": [],
-                "query": "tendrl.clusters.*",
+                "query": "tendrl.name.*",
```

Comment:
Why are we replacing a plural namespace `clusters` with a singular namespace `name`? In my opinion, it should be `names`.

Reply:
@shirshendu Ack! @shtripat can you make the change in sync patch and I will do the change in this patch also.